### PR TITLE
Add WorldEdit CLI Data

### PR DESCRIPTION
**Pull request by me4502**
Adds an API to fetch WorldEdit CLI data, and requires auth tokens for PUT requests.

### CODE REVIEW

File: app/src/main/java/org/enginehub/cassettedeck/controller/CliDataController.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+    /**
+     * Get the WorldEdit CLI data for a data version.
+     *
+     * @return the WE CLI data
+     */
+    @GetMapping("/{dataVersion}")
+    public CliData getWeCliData(
+        @PathVariable int dataVersion,
+        @RequestParam(defaultValue = "1") int cliDataVersion
```
Comment (octylFractal) [inline app/src/main/java/org/enginehub/cassettedeck/controller/CliDataController.java]: What's the CLI data version?

File: app/src/main/java/org/enginehub/cassettedeck/controller/CliDataController.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    public CliDataController(WorldEditCliDataService worldEditCliDataService) {
+        this.worldEditCliDataService = worldEditCliDataService;
+    }
+
+    /**
+     * Get the WorldEdit CLI data for a data version.
+     *
+     * @return the WE CLI data
+     */
+    @GetMapping("/{dataVersion}")
```
Comment (octylFractal) [inline app/src/main/java/org/enginehub/cassettedeck/controller/CliDataController.java]: Why isn't this also `/{dataVersion}/{cliDataVersion}`? That seems more straightforward.

**Review by octylFractal — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/enginehub/cassettedeck/controller/CliDataController.java (hunk 1 above)
Comment (me4502) [reply app/src/main/java/org/enginehub/cassettedeck/controller/CliDataController.java]: It's for if we ever need to change the format of the file itself, as this is a cli data file basically

File: app/src/main/java/org/enginehub/cassettedeck/controller/CliDataController.java (hunk 2 above)
Comment (me4502) [reply app/src/main/java/org/enginehub/cassettedeck/controller/CliDataController.java]: As it's always 1 atm I made it optional, but I'm happy to make it required

**Review by octylFractal — APPROVED** [review]

**Pull request merged**